Pod/Classes/Playback/public/WistiaPlayerViewController.swift

```diff
@@ -46,6 +46,13 @@ public protocol WistiaPlayerViewControllerDelegate : class {
      - Parameter vc: The `WistiaPlayerViewController` whose view is about to appear.
     */
     func willAppear(wistiaPlayerViewController vc: WistiaPlayerViewController)
+    
+    /**
+     Called during at the same time as the `UIKit` standard `ViewController.viewWillDisappear()`.
```

Comment:
😆  D'oh! Thanks. To make this even more meta, I copied this comment from the method above...

I'll fix them both up.